src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -43,34 +43,34 @@ abstract class Particle : WasmObject() {
     open fun onHandleSync(handle: Handle, allSynced: Boolean) {}
 
     fun renderOutput() {
-      log("renderOutput")
-      val slotName = ""
-      val template = getTemplate(slotName)
-      val dict = populateModel(slotName)
-      val model = StringEncoder.encodeDictionary(dict)
-      onRenderOutput(toWasmAddress(), template.toWasmString(), model.toWasmString())
+        log("renderOutput")
+        val slotName = ""
+        val template = getTemplate(slotName)
+        val dict = populateModel(slotName)
+        val model = StringEncoder.encodeDictionary(dict)
+        onRenderOutput(toWasmAddress(), template.toWasmString(), model.toWasmString())
     }
 
     /**
-      * @deprecated for contexts using UiBroker (e.g Kotlin)
-      */
+     * @deprecated for contexts using UiBroker (e.g Kotlin)
+     */
     fun renderSlot(slotName: String, sendTemplate: Boolean = true, sendModel: Boolean = true) {
```

Comment:
If something is deprecated in kotlin,  it's helpful to use the `@Deprecated(reason="", replacement="")` annotation, which will let IntelliJ suggest the replacement for the user.

Reply:
That's a cool language feature.